Data_Engineer_KoNoHa.ipynb

```diff
@@ -0,0 +1,2775 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "provenance": []
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    },
+    "language_info": {
+      "name": "python"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# Data Collection and Scraping"
+      ],
+      "metadata": {
+        "id": "YIgx4fEOW704"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Import Data To Google Drive"
+      ],
+      "metadata": {
+        "id": "g6Px9Xb9XNA9"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "import pandas as pd\n",
+        "from google.colab import drive\n",
+        "\n",
+        "# Mount Google Drive\n",
+        "drive.mount('/content/drive')\n",
+        "\n",
+        "# Define the path to your CSV file\n",
+        "file_path = '/content/drive/MyDrive/data_kaggle.csv'\n",
+        "\n",
+        "# Load the CSV file into a DataFrame\n",
+        "df = pd.read_csv(file_path)"
+      ],
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "_qFLJF0aWjm0",
+        "outputId": "c5d3c968-3bed-4e63-ea36-c29d51569ff9"
+      },
+      "execution_count": null,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "Drive already mounted at /content/drive; to attempt to forcibly remount, call drive.mount(\"/content/drive\", force_remount=True).\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# Data Wrangling"
+      ],
+      "metadata": {
+        "id": "tRXLGdtlLTFu"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Data Exploration"
+      ],
+      "metadata": {
+        "id": "QrHPuEZcQc74"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "# Display the dataframe\n",
+        "df"
+      ],
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 597
+        },
+        "id": "Y6ab8kzRPUrd",
+        "outputId": "541c489e-78d1-4e5b-e648-b7fce64ffd0d"
+      },
+      "execution_count": null,
+      "outputs": [
+        {
+          "output_type": "execute_result",
+          "data": {
+            "text/plain": [
+              "                              Location         Price Rooms  Bathrooms  \\\n",
+              "0                   KLCC, Kuala Lumpur  RM 1,250,000   2+1        3.0   \n",
```

Comment:
Can change location dtypes from object to numeric

Reply:
ok sure can